### DISCUSSION TITLE
IMultiValueConverter

### DISCUSSION THREAD

**Discussion by sunl158** [unoplatform/uno#9912] [Show and tell]
Does IMultiValueConverter is supported?

**Comment by jeromelaban** [comment]
This is a WPF feature that [has not been ported to WinUI](https://github.com/microsoft/microsoft-ui-xaml/issues/3033) yet. Now that custom Markup Wxtensions are supported, it's now possible to implement it, but [x:Bind to functions](https://platform.uno/docs/articles/features/windows-ui-xaml-xbind.html) is also supported.

Could you provide an example of what you're trying to achieve with `IMultiValueConverter ` ?
